va/va.h

```diff
@@ -454,7 +501,7 @@ typedef enum
      * \brief JPEG encoding attribute. Read-only.
      *
      * This attribute exposes a number of capabilities of the underlying
-     * JPEG implementation. The attribute value is partitioned into fields as defined in the 
+     * JPEG implementation. The attribute value is partitioned into fields as defined in the
```

Comment:
Understand, it may cause conflict change when merging other changes in the future, I will change it back.